data/json/construction.json

```diff
@@ -17,6 +17,15 @@
         "post_special" : "done_deconstruct"
     },{
         "type" : "construction",
+        "description" : "Take Down Simple Furniture",
+        "category" : "FURN",
+        "required_skills" : [ [ "fabrication", 0 ] ],
+        "time" : 10,
+        "pre_flags" : "EASY_DECONSTRUCT",
+        "pre_special" : "check_deconstruct",
+        "post_special" : "done_deconstruct"
```

Comment:
Might be good then, I will look at the feature to see how to apply it. Could apply these notes to the terrain and furniture on examine, as well. If that is possible?